src/components/base/ResponsiveButton.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<div>
+		<mq-layout mq="tablet+">
+			<base-button :design="design" v-on="$listeners">
+				<base-icon :source="source" :icon="icon" />
+				<slot />
+			</base-button>
+		</mq-layout>
+		<mq-layout mq="mobile">
+			<base-button :design="icondesign" v-on="$listeners">
+				<base-icon :source="source" :icon="icon" />
```

Comment:
```suggestion
				<base-icon v-bind="$props"/>
```

auch hier v-bind props... dann geht das icon und source automatisch durch